pkg/model/resources/nodeup.go

```diff
@@ -177,6 +184,7 @@ type NodeUpScript struct {
 	NodeUpAssets         map[architectures.Architecture]*mirrors.MirroredAsset
 	KubeEnv              string
 	CompressUserData     bool
+	RestartJournald      bool
```

Comment:
Maybe just pass the CloudProvider instead? I am sure there will be other use cases for it.

Reply:
sure, updated.